.circleci/config.yml

```diff
@@ -476,6 +476,19 @@ jobs:
       - store_test_results:
           path: e2e-tests/trailing-slash/cypress/results
 
+  e2e_tests_gatsby_script:
+    <<: *e2e-executor
+    environment:
+      <<: *e2e-executor-env
+      CYPRESS_PROJECT_ID: 3g7nyg
+      CYPRESS_RECORD_KEY: f8007925-dc92-4136-9e92-8b0842d54719
+    steps:
+      - e2e-test:
+          test_path: e2e-tests/gatsby-script
+          test_command: yarn test
+      - store_test_results:
+          path: e2e-tests/gatsby-script/cypress/results
+
```

Comment:
Seems like we do need to actually add this job to `build-test` workflow https://github.com/gatsbyjs/gatsby/blob/ca4eb059beaa62138a6a1a5b2fb2e9c8960001df/.circleci/config.yml#L564-L589

As currently we have job defined but never executed it seems